app/src/main/java/com/example/bankingapplication/presentation/screens/RecentTransactionsScreen.kt

```diff
@@ -153,9 +153,9 @@ fun RecentTransactionsScreen(
                     fontWeight = FontWeight.Bold,
                     fontSize = 28.sp
                 )
-                AccountCard{}
-                AccountCard{}
-                AccountCard{}
+                AccountCard {}
+                AccountCard {}
+                AccountCard {}
```

Comment:
what is that?

Reply:
idk, android studio added these spaces after reformat code (pressing alt+shift+L)